cname_records.php

```diff
@@ -39,6 +39,10 @@
                 </div>
             </div>
             <div class="box-footer clearfix">
+              <strong>Note:</strong>
+                  <p>The target of a <code>CNAME</code> must be a domain that the Pi-hole knows the <code>A</code> value for, or is authoritative for that domain.</p>
+                  <p>Pi-hole will not do additional queries if you set the target to a domain that isn't already known. It will return the information it knows at the time of the query,
+                    without having to send additional queries to an upstream for resolution. That means, for example, target domains that exist as DHCP leases only will not be fully resolved until you specify an explicit DNS entry.</p>
```

Comment:
I'd add something about external domains not working either. You can't CNAME `bing.com` to `google.com`.

Reply:
Somthing like

"Additionally, you can't `CNAME` external domains (`bing.com` to `google.com`) successfully as this would result in an invalid SSL certificate error."

Reply:
Yep, that sounds good to me.